doc/source/data/dataset.rst

```diff
@@ -2,48 +2,55 @@
-Ray Data: Distributed Data Loading and Compute
-==============================================
+==================================================
+Ray Datasets: Distributed Data Loading and Compute
+==================================================
 
-Ray Data is a library for loading and processing large datasets.
-Its key abstraction is that of a Dataset.
-Datasets are the standard way to load and exchange data in Ray libraries and applications.
+Ray Datasets are the standard way to load and exchange data in Ray libraries and applications.
 They provide basic distributed data transformations such as ``map``, ``filter``, and ``repartition``,
 and are compatible with a variety of file formats, data sources, and distributed frameworks.
 
 Here's an overview of the integrations with other processing frameworks, file formats, and supported operations,
-as well as glimpse at the Ray Data API.
+as well as glimpse at the Ray Datasets API.
 Check our :ref:`compatibility matrix<data-compatibility>` to see if your favorite format is supported already.
 
 .. image:: images/dataset.svg
 
 ..
   https://docs.google.com/drawings/d/16AwJeBNR46_TsrkOmMbGaBK7u-OPsf_V8fHjU-d2PPQ/edit
 
-Ray Data simplifies general purpose parallel GPU and CPU compute in Ray,
-for instance for `GPU batch inference <dataset.html#transforming-datasets>`__).
+Ray Datasets simplifies general purpose parallel GPU and CPU compute in Ray; for instance, for `GPU batch inference <dataset.html#transforming-datasets>`__.
 It provides a higher level API for Ray tasks and actors in such embarrassingly parallel compute situations,
 internally handling operations like batching, pipelining, and memory management.
 
 .. image:: images/dataset-compute-1.png
    :width: 500px
    :align: center
 
-As part of the Ray ecosystem, Ray Data can leverage the full functionality of Ray's distributed scheduler,
+As part of the Ray ecosystem, Ray Datasets can leverage the full functionality of Ray's distributed scheduler,
 e.g., using actors for optimizing setup time and GPU scheduling.
 
+----------------------------------------------
+Data Loading and Preprocessing for ML Training
+----------------------------------------------
+
+Ray Datasets are designed to load and preprocess data for distributed :ref:`ML training pipelines <train-docs>`.
+Compared to other loading solutions, Datasets are more flexible (e.g., can express higher-quality `per-epoch global shuffles <examples/big_data_ingestion.html>`__) and provides `higher overall performance <https://www.anyscale.com/blog/why-third-generation-ml-platforms-are-more-performant>`__.
+
+Ray Datasets is not intended as a replacement for more general data processing systems.
+:ref:`Learn more about how Ray Datasets works with other ETL systems <datasets-ml-preprocessing>`.
+
 ----------------------
 Where to Go from Here?
 ----------------------
 
-As new user of Ray Data, you may want to start with our :ref:`Getting Started Guide<data_getting_started>`.
```

Comment:
On this one I slightly disagree. The whole navigation item is called "Ray Data". In this whole document we're not using this formulation once. How are users supposed to know what "Ray Data" is then?

I think at the top we need an intro sentence like "Ray Data is a collection of Dataset APIs and integrations (e.g. link to bla)". We can't just assume they kind of get what we intend, we have to state it clearly. With a better intro I think it's then fair to say that if you want to get started with Ray Data (not Datasets) you should check out the Ray Datasets starting guide.

Reply:
I was hoping to revert everything to "Ray Datasets" in this PR and then create the "Ray Data" ecosystem overview in a follow-up PR, once I've had time to generate the content. My highest priority right now is making sure that (prospective) Datasets users don't get confused by the mixed branding of the library, so I'm trying to disentangle the "Ray Data" ecosystem overview content review from the branding revert review so I can get the latter in more quickly.

Reply:
ok, makes sense.